src/Microsoft.AspNetCore.Server.Testing/Deployers/ApplicationDeployer.cs

```diff
@@ -17,95 +17,48 @@ namespace Microsoft.AspNetCore.Server.Testing
-                ToolingRuntimeBinPath = TargetRuntimeBinPath;
-            }
-            else
-            {
-                var flavor = DeploymentParameters.RuntimeFlavor == RuntimeFlavor.CoreClr ? "coreclr" : "clr";
-                var architecture = DeploymentParameters.RuntimeArchitecture == RuntimeArchitecture.x86 ? "x86" : "x64";
-
-                // tooling runtime will stick to coreclr so as to prevent long path issue during publishing
-                ToolingRuntimeBinPath = Path.Combine(runtimeHome, $"dnx-coreclr-{GetOSPrefix()}-{architecture}.{currentRuntimeVersion}", "bin");
-
-                TargetRuntimeName = $"dnx-{flavor}-{GetOSPrefix()}-{architecture}.{currentRuntimeVersion}";
-                TargetRuntimeBinPath = Path.Combine(runtimeHome, TargetRuntimeName, "bin");
-            }
-
-            if (!Directory.Exists(ToolingRuntimeBinPath) || !Directory.Exists(TargetRuntimeBinPath))
-            {
-                throw new Exception($"Requested runtime '{ToolingRuntimeBinPath}' or '{TargetRuntimeBinPath}; does not exist. Please make sure it is installed before running test.");
-            }
-
-            Logger.LogInformation($"Pick target runtime {TargetRuntimeBinPath}");
-            Logger.LogInformation($"Pick tooling runtime {ToolingRuntimeBinPath}");
-
-            // Work around win7 search path issues.
-            var newPath = TargetRuntimeBinPath + Path.PathSeparator + Environment.GetEnvironmentVariable("PATH");
-            DeploymentParameters.EnvironmentVariables.Add(new KeyValuePair<string, string>("PATH", newPath));
+            Logger.LogInformation($"Pick target frameowkr {TargetFrameworkName}");
         }
 
-        protected void DnuPublish(string publishRoot = null)
+        protected void DotnetPublish(string publishRoot = null)
         {
             DeploymentParameters.PublishedApplicationRootPath = publishRoot ?? Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
 
-            var noSource = DeploymentParameters.PublishWithNoSource ? "--no-source" : string.Empty;
-            var command = DeploymentParameters.Command ?? "web";
-            var parameters = $"publish {DeploymentParameters.ApplicationPath} -o {DeploymentParameters.PublishedApplicationRootPath}"
-                + $" --runtime {TargetRuntimeName} {noSource} --iis-command {command}";
+            var parameters = $"publish {DeploymentParameters.ApplicationPath}"
+                + $" -o {DeploymentParameters.PublishedApplicationRootPath}"
+                + $" --framework {TargetFrameworkName}";
 
-            var dnuPath = Path.Combine(ToolingRuntimeBinPath, DnuCommandName);
-            Logger.LogInformation($"Executing command {dnuPath} {parameters}");
+            Logger.LogInformation($"Executing command {DotnetCommandName} {parameters}");
 
             var startInfo = new ProcessStartInfo
             {
-                FileName = dnuPath,
+                FileName = DotnetCommandName,
```

Comment:
I was surprised when I saw the exception thrown on line 79 which says "dnu publish exited...", that should be updated.